spec/Appendix A -- Persisted Documents.md

```diff
@@ -95,6 +95,35 @@ implementation specific.
 Note: A 32 character hexadecimal _custom document identifier_ is likely to be an
 MD5 hash of the GraphQL document, as traditionally used by Relay.
 
+### Document identifier syntax
+
+DocumentIdentifier ::
+
+- PrefixedDocumentIdentifier
+- CustomDocumentIdentifier
+
+PrefixedDocumentIdentifier ::
+
+- Sha256HexDocumentIdentifier
+- OtherPrefixedDocumentIdentifier
+
+Sha256HexDocumentIdentifier : `sha256:` Sha256Checksum
+
+Sha256Checksum :: 64 LowerCaseHexDigit
```

Comment:
Can you point to anything in spec-md or the GraphQL spec that uses this syntax to say "64 of this thing"? I'd expect something more like `LowerCaseHexDigit{64}` but I had a quick scan last week and didn't see anything about specific repetition counts. In the end I decided to got be as strict on the _syntax_ of identifiers and leave that for _validation_ - the errors you get out of validation can be a lot more helpful than the ones you get from syntax.

Reply:
> Can you point to anything in spec-md or the GraphQL spec that uses this syntax to say "64 of this thing"? 

This is completely made up. I don't think we have anything like that at the moment.

> In the end I decided to not be as strict on the syntax of identifiers and leave that for validation

Works for me 👍 , I'll amend the PR

Reply:
https://github.com/graphql/graphql-over-http/pull/296/commits/3fbc34c7dac239eaa6ec3778e13abfe34fd3e077